docs/sections/data/groups.md

```diff
@@ -168,65 +149,57 @@ The same happens if you remove nodes from the group -- they will remain in the d
 ## Organising groups in hierarchies
 
 Earlier we mentioned that groups are like files in folders on your filesystem.
-As with folders and sub-folders then, as the amount of groups we have grows, we may also wish to structure our groups in a hierarchy.
-Groups in AiiDA are inherently "flat", in that groups may only contain nodes and not other groups.
-However it is possible to construct *virtual* group hierarchies based on delimited group labels, using the `grouppath` utility.
+Then as with folders and sub-folders, as the amount of groups you have grows, you may also wish to structure your groups in a hierarchy.
+Groups in AiiDA are inherently "flat", meaning groups may only contain nodes and not other groups.
+However, it is possible to construct *virtual* group hierarchies based on delimited group labels, using the `grouppath` utility.
 
-Like folder paths grouppath requires delimitation by `/` (forward slash) characters.
+Like folder paths on Unix systems `grouppath` requires delimitation by forward slash (`/`) characters.
 Lets copy and rename the three tutorial groups:
 
 ```{code-block} console
-
-$ verdi group copy tutorial_lda tutorial/lda/basic
-$ verdi group copy tutorial_pbe tutorial/gga/pbe
-$ verdi group copy tutorial_pbesol tutorial/gga/pbesol
-
+verdi group copy tutorial_lda tutorial/lda/basic
+verdi group copy tutorial_pbe tutorial/gga/pbe
+verdi group copy tutorial_pbesol tutorial/gga/pbesol
 ```
 
-We can now list the groups in a new way:
+You can now list the groups in a new way:
 
 ```{code-block} console
-
 $ verdi group path ls -l
 Path             Sub-Groups
 ---------------  ------------
 tutorial                    3
 tutorial_lda                0
 tutorial_pbe                0
 tutorial_pbesol             0
-
 ```
 
 :::{note}
 
-In the terminal, paths that contain nodes are listed in bold
+In the terminal, paths that contain nodes are listed in bold.
```

Comment:
Is this true? I don't seem to notice this. 🤔

Reply:
I can't actually check it now, but the code has `bold=True`? https://github.com/aiidateam/aiida-core/blob/develop/aiida/cmdline/commands/cmd_group.py#L448

Reply:
Well, i'll double check it during my pass, thanks!